.github/workflows/canary_release.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Release Canary'
+on:
+  push:
+    branches:
+      - '4.x'
+
+jobs:
+  release:
+    name: release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: yarn
+      - run: yarn install --ignore-scripts
+      - run: yarn prebuild
+      - run: yarn build:cjs      
+      - run: yarn build:esm
+      - run: yarn publish:canary
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+      
```

Comment:
Could you also update https://github.com/web3/web3.js/blob/4.x/RELEASE.md Thanks

Reply:
Hm, what do you want me to add there? Isn't that a guide for cutting a full release mannually?

Reply:
I think under `Version Number Definition` , we should add `dev` ?

Reply:
But I'm not introducing new version number definition, dev is just npm tag of the version:
![image](https://github.com/web3/web3.js/assets/8836210/ae4ecc3a-2cba-4f15-8401-98adb18b3b67)